doc/source/operations/tempest.rst

```diff
@@ -0,0 +1,303 @@
+======================================
+Running Tempest with Kayobe Automation
+======================================
+
+Overview
+========
+
```

Comment:
Should mention that the "proper" way to do this is using a CI/CD workflow, but this is for running manually

Reply:
Done